R/run-nmtran.R

```diff
@@ -0,0 +1,375 @@
+#' Interface for running `NM-TRAN` on model objects
+#'
+#' Function to run `NM-TRAN` on a model object to validate its control stream
+#'  for correct coding before submission. The `NM-TRAN` dataset (`FDATA`) and
+#'  other `NONMEM` artifacts can be further inspected by keeping the run directory
+#'  around.
+#'
+#' @details
+#' `NM-TRAN` is a preprocessor for `NONMEM` that translates user-specified
+#' control stream data and instructions into a form executable by `NONMEM`.
+#'
+#' Note that `nmtran_presort` is run ahead of `NM-TRAN` for `NONMEM 7.4` and later
+#'  - `nmtran_presort` is a supplementary utility that preprocesses the control
+#'     stream to ensure it is in the correct format for `NM-TRAN`.
```

Comment:
[ following on [previous thread](https://github.com/metrumresearchgroup/bbr/pull/705#discussion_r1693610272) ]

I still don't feel confident that we can say why nmfe runs `nmtran_presort`, so the "preprocess the control stream to ensure it is in the correct format" is a bit more than I think we should say.

But the claims about what `nmtran_presort` does and why it exists is weakened from the previous version. This is good enough for me.